src/package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "zarrita",
+  "version": "0.1.5",
+  "description": "A minimal, exploratory implementation of the Zarr version 3.0 core protocol.",
+  "type": "module",
+  "scripts": {
+    "prepack": "cp ../README.md ."
+  },
+  "sideEffects": [
+    "zarrita.js"
+  ],
+  "files": [
+    "**/*.js"
+  ],
+  "exports": {
+    ".": "./zarrita.js",
+    "./core": "./core.js",
+    "./lib/*": "./lib/*.js",
+    "./storage/httpstore": "./storage/httpstore.js",
+    "./storage/fsstore": {
+      "node": "./storage/fsstore.js"
```

Comment:
`"node"` flag will ensure that CDNs like Skypack won't try to build this entry point.